content/blog/2024-02-06-tmix-01.md

```diff
@@ -0,0 +1,24 @@
++++
+title = "This Month in Xilem, January 2024"
+authors = ["Raph Levien"]
+date = "2024-02-06 07:28:42"
++++
+
+Welcome to the first in the “this month in Xilem” series, bringing updates on the [Xilem] UI toolkit.
+
+There were two blog posts on the new [linebender blog](https://linebender.org/blog).
+The first, [Xilem 2024 plans](https://linebender.org/blog/xilem-2024/) sets out the goals for the project for the year.
```

Comment:
I don't know how to make it flow, but it might be worth noting the author of this post as well